api/include/opentelemetry/trace/trace_flags.h

```diff
@@ -31,13 +31,16 @@ class TraceFlags final
 {
 public:
   static constexpr uint8_t kIsSampled = 1;
+  static constexpr uint8_t kForbidden = 0xFF;
 
   TraceFlags() noexcept : rep_{0} {}
 
   explicit TraceFlags(uint8_t flags) noexcept : rep_(flags) {}
 
   bool IsSampled() const noexcept { return rep_ & kIsSampled; }
 
+  bool IsValid() const noexcept { return rep_ != kForbidden; }
```

Comment:
Where is it specified that `0xff` is an invalid value for trace state?

Reply:
Ah my bad, thanks! 0xFF was an invalid marker for the version not trace flags. Will fix in the morning, not at the computer atm

Reply:
Fixed - it now checks the version for 0xFF not trace flags :facepalm: